### ISSUE TITLE
Circular reference not supported with oneOf and array

### ISSUE THREAD

**Issue by claudiocantalupo-spotsoftware** [horejsek/python-fastjsonschema#42]
Hello,
I found this weird bug in compile code generation; to reproduce just create a file named Section.json with the following content (create another dummy json for OtherSection too).

```
{
  "title": "Section",
  "type": "object",
  "properties": {
    "SectionList": {
      "oneOf": [
        {
          "type": "array",
          "items": {
            "$ref": "file:///Section.json"
          }
        },
        {
          "type": "array",
          "items": {
            "$ref": "file:///OtherSection.json"
          }
        }
      ]
    }
  }
}
```

The combination of the 'oneOf' and 'array' seems to drive crazy the code generator which raises the exception _IndentationError: expected an indented block_.

Thanks for this great package!

**Comment by horejsek** [comment]
Thanks for the report, will look at it in next few days.

**Comment by horejsek** [comment]
Can you provide failing example? When I use the combination with latest version, it's working.

**Comment by claudiocantalupo-spotsoftware** [comment]
I am not able to reproduce it anymore; maybe something messed up in my virtualenv yesterday because I have tried dozen of times and the compile *always* returned _IndentationError: expected an indented block_. Looking at the compiled code the portion to manage the array inside the oneof tag was badly indentated, hence the above error

Today I have restarted the virtualenv (Python 3.6.7 on Win64) and now the problem has gone.
If the problem occurs again I will try to isolate it better.

Thanks a lot.
